DSP Accel Library/Testing and Images/testingTEMP.m

```diff
@@ -0,0 +1,52 @@
+
+%% Before Running
+x_in = 0:0.001:3;
+
+tt = 0:length(x_in)-1;
+tt = tt./48000;
+
+simin_Data_In = [tt',x_in'];
+simin_Table_Wr_Data = [tt',x_in'];
+simin_Table_Wr_Addr = [tt',x_in'];
+simin_Wr_En = [tt',x_in'];
+stop_time = tt(end);
+
+
+
+%% After Running
+
+output = out.simout.Data(:);
+outputIdeal = sqrt(x_in);
+address = addr.Data(:);
+%address(2:1001,1) = address(1:1000,1);
+%address(:,1) = abs(31-address(:,1));
+%address_r = address(:,1).*2+address(:,2);
+address(1:3000) = address(2:3001);
```

Comment:
It would be useful to add a couple more descriptive variables to the script instead of using the numbers directly in the address field.  Also, a comment about why this is happening would help.